.azure-pipelines/templates/variables.yml

```diff
@@ -1,6 +1,6 @@
 variables:
-  ubuntu_pool: 'pool-ubuntu-2004'
-  ubuntu_multi_core_pool: 'pool-ubuntu-2004-multi-core'
+  ubuntu_pool: 'pool-ubuntu-2204'
```

Comment:
The reason for not using `pool-ubuntu-latest` is that Azure PowerShell team has already created `pool-ubuntu-2204`.